typegate/tests/prisma/prisma_edgecases.py

```diff
@@ -0,0 +1,30 @@
+from typegraph import TypeGraph, effects, policies, t
+from typegraph.providers.prisma.runtimes.prisma import PrismaRuntime
+
+with TypeGraph("prisma") as g:
+    # schema ref:
+    # https://www.prisma.io/docs/reference/api-reference/prisma-client-reference
+
```

Comment:
For later pull requests, try to come up with addition to current testing typegraphs instead of creating new ones as it might introduce a cost when we have to refactor something on all of them.

Reply:
Sure, as for a reason, I created a new tg for simplification and to isolate the issue, all the typegraphs seemed too heavy and unrelated. I also expected `min(.).max(.)` schema issue to be a bug on our side thus potentially requiring further edgecase testings.

Reply:
`findFirst` is not yet implemented but it should be easy to add support for it.

Reply:
Great, can you include it in this PR ?

Reply:
no problem 👍